src/i965_drv_video.c

```diff
@@ -2766,6 +2776,10 @@ i965_DestroyContext(VADriverContextP ctx, VAContextID context)
         obj_context->wrapper_context = VA_INVALID_ID;
     }
 
+    drm_intel_gem_context_destroy(i965->gem_context);
+
```

Comment:
drm_intel_bufmgr_destroy() should be called to release i965->bufmgr.

Reply:
Ok, I'll add that and retest.

Reply:
@xhaihao please take a look at updated patch on this branch.